depends/packages/fontconfig.mk

```diff
@@ -11,6 +11,7 @@ define $(package)_set_vars
 endef
 
 define $(package)_config_cmds
+  export PKG_CONFIG_PATH=$(host_prefix)/lib/pkgconfig && \
```

Comment:
This shouldn't be needed, it's already done by $($(package)_autoconf). Was there an issue that prompted these additions?